src/components/api.js

```diff
@@ -0,0 +1,149 @@
+export default class Api {
+  constructor({ baseUrl, authToken }) {
+    this._baseUrl = baseUrl;
+    this._authToken = authToken;
+  }
+
+  getCardList() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: {
+        authorization: this._authToken,
+      },
+    })
+      .then((res) =>
+        res.ok ? res.json() : Promise.reject(`Error: ${res.status}`)
+      )
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  addCard({ name, link }) {
+    return fetch(`${this._baseUrl}/cards`, {
+      method: "POST",
+      headers: {
+        authorization: this._authToken,
+        "Content-Type": "application/json",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)